.devcontainer/install_avalanchego.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
we already have a script in here: https://github.com/ava-labs/precompile-evm/blob/main/scripts/install_avalanchego_release.sh. can we use it?